create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/wait_script.py

```diff
@@ -1,17 +1,30 @@
-{%- if cookiecutter.use_postgres == 'y' %}
-
-import socket
+{%- if cookiecutter.use_postgres == 'y' or cookiecutter.use_redis == 'y' %}import socket
```

Comment:
Rather than shipping empty module I'd use hooks to remove if unneeded.

Reply:
The module is removed if neither postgres, nor redis is used in the project